### ISSUE TITLE
Negative floating point constants push mantissa out of view

### ISSUE THREAD

**Issue by inscripttech** [kometbomb/prototracker-modular#42]
When using a negative floating point constant, the minus symbol pushes the part of the number after the decimal out of view.

**Comment by cupcake99** [comment]
I have a sort of fix for this issue in my fork on the 'wilderness' branch which i use to experiment. It changes the colour of the text and adds a little negative symbol in the corner of the module. But i only implemented it in the display module so far, i forgot to transfer the change through to the const module. Wouldn't be difficult though. Only thing is that the text doesn't quite fit the bounds of the box any more, which is kind of ugly in its own way.

Looks like this:

![Screen Shot 2020-05-28 at 02 23 48](https://user-images.githubusercontent.com/13849330/83087482-fb43bd00-a080-11ea-83a2-0e60f28a9678.png)

**Comment by inscripttech** [comment]
Yeah, makes sense. I hope posting this as an issue isn't too much of a nitpick.

Another idea may be to replace the decimal with the negative but also not elegant.

**Comment by kometbomb** [comment]
More ideas: 
* a higher res UI (so the font can be more delicate)
* a smaller font for numbers
